src/main/kotlin/no/nav/familie/ef/sak/vedtak/dto/VedtakDto.kt

```diff
@@ -97,8 +119,13 @@ fun Vedtak.tilVedtakDto(): VedtakDto =
                     begrunnelse = this.avslåBegrunnelse,
                     opphørFom = YearMonth.from(this.opphørFom)
             )
+            ResultatType.SANKSJONERE -> Sanksjonert(
+                    resultatType = this.resultatType,
+                    sanksjonsårsak = this.sanksjonsårsak ?: Sanksjonsårsak.INGEN_VALGT,
+                    periode = (this.perioder?.perioder ?: PeriodeWrapper(emptyList()).perioder).fraDomeneForSanksjon(),
+                    internBegrunnelse = this.internBegrunnelse ?: "",
```

Comment:
Burde denne kunne være null i stedet for å sette tom string? Ser de andre begrunnelsene er nullable? Og finnes kanskje en risiko att frontend sender inn null i stedet for tom string?